pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/api/FunctionsImpl.java

```diff
@@ -617,7 +608,7 @@ public Response uploadFunction(final @FormDataParam("data") InputStream uploaded
             Utils.uploadToBookeeper(
                     worker().getDlogNamespace(),
                     uploadedInputStream,
-                    path);
+                    Codec.encode(path));
```

Comment:
If we are doing this for upload, then we need a corresponding thing done in the download as well.

Reply:
nevermind, I didn't see the change below